modules/core/ui/metrics/src/Plugin/Block/FarmMetricsBlock.php

```diff
@@ -74,7 +74,7 @@ public function build() {
 
     // Create a container for asset metrics.
     $output['asset'] = [
-      '#markup' => '<strong>' . Link::createFromRoute('Assets', 'view.farm_asset.page')->toString() . '</strong>',
+      '#markup' => '<strong>' . Link::createFromRoute($this->t('Assets'), 'view.farm_asset.page')->toString() . ' (' . $this->t('active') . ')</strong>',
```

Comment:
Oh sorry... didn't remember this earlier. Should we use the entity "collection" label?

https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21EntityType.php/function/EntityType%3A%3AgetCollectionLabel/9.2.x

Was thinking back to this: https://github.com/farmOS/farmOS/pull/458#discussion_r747697108